hack/jenkins/e2e.sh

```diff
@@ -192,6 +192,19 @@ case ${JOB_NAME} in
     MINION_SIZE="n1-standard-1"
     NUM_MINIONS="2"
     ;;
+
+  # Runs non-flaky tests on GCE on the release-latest branch,
+  # sequentially. As a reminder, if you need to change the skip list
```

Comment:
oh right, you mentioned that this job will be used along with the build job.

maybe clarify exactly what that means in the comment, then? maybe also set KUBE_RUN_FROM_OUTPUT=y here?